spock/mcp/nbt.py

```diff
@@ -470,10 +514,15 @@ def valuestr(self):
     def pretty_tree(self, indent=0):
         output = [super(TAG_Compound, self).pretty_tree(indent)]
         if len(self.tags):
-            output.append(("\t"*indent) + "{")
+            output.append(("\t" * indent) + "{")
             output.extend([tag.pretty_tree(indent + 1) for tag in self.tags])
-            output.append(("\t"*indent) + "}")
+            output.append(("\t" * indent) + "}")
         return '\n'.join(output)
 
 
-TAGLIST = {TAG_END: _TAG_End, TAG_BYTE:TAG_Byte, TAG_SHORT:TAG_Short, TAG_INT:TAG_Int, TAG_LONG:TAG_Long, TAG_FLOAT:TAG_Float, TAG_DOUBLE:TAG_Double, TAG_BYTE_ARRAY:TAG_Byte_Array, TAG_STRING:TAG_String, TAG_LIST:TAG_List, TAG_COMPOUND:TAG_Compound, TAG_INT_ARRAY:TAG_Int_Array}
+TAGLIST = {TAG_END: _TAG_End, TAG_BYTE: TAG_Byte, TAG_SHORT: TAG_Short,
+           TAG_INT: TAG_Int, TAG_LONG: TAG_Long,
+           TAG_FLOAT: TAG_Float, TAG_DOUBLE: TAG_Double,
+           TAG_BYTE_ARRAY: TAG_Byte_Array, TAG_STRING: TAG_String,
+           TAG_LIST: TAG_List, TAG_COMPOUND: TAG_Compound,
+           TAG_INT_ARRAY: TAG_Int_Array}
```

Comment:
As you wish, Will have a look on it later, as probably flake with plugins is saying something about this, but yeah, looks better like that